src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -237,6 +271,16 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
         );
     }
 
+    setLinkFromUDW(attrName, item) {
+        const values = Object.assign({}, this.state.values);
```

Comment:
In theory you shouldn't rely on `this.state` when modyfing state, ref: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous